e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -532,7 +532,7 @@ describe("#39152 sharing an unsaved question", () => {
                 titled: true,
                 font: "instance",
                 theme: "light",
-                hide_download_button: resource === "question" ? false : null,
```

Comment:
I'm temporarily changing the tests to expect null, i'll update the analytics + tests in the next MS to not keep this open too much